src/lib/ngsi/Request.cpp

```diff
@@ -75,6 +75,7 @@ const char* requestType(RequestType rt)
   case Ngsi10SubscriptionsConvOp:                   return "Ngsi10SubscriptionsConvOp";
 
   case LogRequest:                                  return "Log";
+  case LogLevelRequest:                             return "LogLevel";
   case VersionRequest:                              return "Version";
```

Comment:
So, https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/management_api.md is correct or wrong?